### ISSUE TITLE
Tabs Bar Disappears After Clicking in the Editor

### ISSUE THREAD

**Issue by Rhindon** [Scirra/Construct-3-bugs#4459]
## Problem description

After initially loading the C3 editor, the tabs bar - Start, Layouts, Event Sheets - will disappear entirely after clicking virtually anywhere in the editor. I have determined, thus far, that it doesn't seem limited to clicking on any specific areas (such as clicking in other areas may not cause the bar to disappear). Further, double-clicking on a Layout or Event Sheet listing in the Project Bar window does not cause the tabs bar to reappear.

I re-tested it again... I clicked on the initial Event Sheet tab and the Event Sheet loaded. Then I clicked on the Layout tab and the tabs bar disappeared. I have not been able to narrow down the cause or what is or is not part of the problem beyond that.

## Attach a .c3p

I've attached a ZIP of the most recent C3 project I had been working on prior to the problem occurring. However, this happens even when starting a new project, as the attached MP4 shows.

[Anti_Virus alpha 0-2 crash recovery zip.zip](https://github.com/Scirra/Construct-3-bugs/files/5790271/Anti_Virus.alpha.0-2.crash.recovery.zip.zip)

https://user-images.githubusercontent.com/12901011/104085575-871beb00-5205-11eb-8476-3d11b81a8414.mp4

## Steps to reproduce

1. Start C3 anew and start a new project or open an existing one. Make sure the release version is at least r225 (it might go back further than that but it occurs in even the most recent update).
2. When the project is loaded, click virtually anywhere in the editor.
3. Observe the tabs bar as it will likely disappear.

## Observed result

The tabs bar completely disappears when the mouse is clicked (almost) anywhere in the editor.

## Expected result

The tabs bar to remain visible and accessible.

## More details

I cannot tell if this may have anything to do with the browser as I do not recall any updates to Chrome in the last couple of months and I haven't closed Chrome (I usually leave my computer running when I step away) and have not seen a notification for updates.

**Affected browsers/platforms:** 
Currently, only Chrome. I do not use any other browsers.

**First affected release:** 
r225 is the earliest I recall seeing the problem.
r230 is the most recent version I've used.
I had not been working on my project during releases 226-229 but it's likely the problem was present then, too.

## System details

<details><summary>View details</summary>

No crash occurred.

</details>

**Comment by AshleyScirra** [comment]
It seems to work fine for me, and I have also never seen any other reports of this from anyone else so far.

Do you have any browser extensions installed? Try disabling them, in case one of them is interfering.

Please also attach your system details as per the bug report guidelines. We always need it, even if there is no crash.

**Comment by Rhindon** [comment]
AH, sorry about that. Below is a copy of the system/platform info as collected from the About link in C3.

I went ahead and disabled/removed all extensions on Chrome and even restarted my laptop for good measure. I'm seeing the same issue. If it helps at all, I closed out the new project I started and the Start tab and its respective bar reappeared. So, it seems limited to an active project. The weird thing is that I've been using C3 for months on this particular laptop and I'm unsure what I did differently (new program? Changed a setting?) that could have caused this. Certainly nothing related that I can think of.

I have a brand new laptop on order that should be arriving at the end of the month. I'll get things set up anew and try to reproduce the issue and maybe that might shed some light on things...especially if you can't find the problem on your end sooner.

Anyway, thanks for looking into this, Ashley. I always appreciate your work and all you and your team do for us.

=======

Platform information
Browser: Chrome
Browser version: 87.0.4280.88
Browser engine: Chromium
Browser architecture: 64-bit
Context: webapp
Operating system: Windows
Operating system version: 10
Operating system architecture: 64-bit
Device type: desktop
Device pixel ratio: 1
Logical CPU cores: 4
Approx. device memory: 8 GB
User agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Safari/537.36
C3 release: r230 (beta)
Language setting: en-US

Local storage
Storage quota (approx): 142 gb
Storage usage (approx): 294 mb (0.2%)
Persistant storage: Yes

Browser support notes
This list contains missing features that are not required, but could improve performance or user experience if supported.

Nothing is missing. Everything is OK!
WebGL information
Version string: WebGL 2.0 (OpenGL ES 3.0 Chromium)
Numeric version: 2
Supports NPOT textures: yes
Supports GPU profiling: yes
Supports highp precision: yes
Vendor: Google Inc.
Renderer: ANGLE (Intel(R) HD Graphics 520 Direct3D11 vs_5_0 ps_5_0)
Major performance caveat: no
Maximum texture size: 16384
Point size range: 1 to 1024
Extensions:

EXT_color_buffer_float
EXT_color_buffer_half_float
EXT_disjoint_timer_query_webgl2
EXT_float_blend
EXT_texture_compression_bptc
EXT_texture_compression_rgtc
EXT_texture_filter_anisotropic
EXT_texture_norm16
KHR_parallel_shader_compile
OES_texture_float_linear
WEBGL_compressed_texture_s3tc
WEBGL_compressed_texture_s3tc_srgb
WEBGL_debug_renderer_info
WEBGL_debug_shaders
WEBGL_lose_context
WEBGL_multi_draw
OVR_multiview2
Audio information
System sample rate: 48000 Hz
Output channels: 2
Output interpretation: speakers
Supported decode formats:

WebM Opus (audio/webm; codecs=opus)
Ogg Opus (audio/ogg; codecs=opus)
WebM Vorbis (audio/webm; codecs=vorbis)
Ogg Vorbis (audio/ogg; codecs=vorbis)
MPEG-4 AAC (audio/mp4; codecs=mp4a.40.5)
MP3 (audio/mpeg)
FLAC (audio/flac)
PCM WAV (audio/wav; codecs=1)
Supported encode formats:

WebM Opus (audio/webm; codecs=opus)
Video information
Supported decode formats:

WebM AV1 (video/webm; codecs=av01.0.00M.08)
MP4 AV1 (video/mp4; codecs=av01.0.00M.08)
WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)
Ogg Theora (video/ogg; codecs=theora)
H.264 (video/mp4; codecs=avc1.42E01E)
Supported encode formats:

WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)

**Comment by SnipG** [comment]
Easy way to determine if it's chrome bug. Just use Microsoft edge browser. If so, you would need to reinstall chrome.

**Comment by AshleyScirra** [comment]
I tried using as close to the same setup as possible (Chrome 87 on Windows 10 installed as an app) and I can't reproduce it at all. It seems to be working just fine.

I'm afraid without a reliable way to reproduce the issue it won't be possible to do anything about this. Can you reproduce it on any other systems or find any other significant details about how to reproduce it?

**Comment by Rhindon** [comment]
I'll see what I can do later today after work.
As mentioned, I have a brand new laptop coming my way at the end of the month and I'll see if the issue persists...
Thank you for bearing with me as I'm sure this is confusing as it is.

**Comment by Rhindon** [comment]
Well...
Let's all scratch our heads until we're bald because that was certainly an unsolved mystery.
However, the suggestion by SnipG worked - I uninstalled and reinstalled Chrome and everything's working.
I did test C3 in MS Edge and, as expected, it did work properly.
SO...! Problem "solved"? LOL

In the very off-chance that this will help you discern the issue, I have copied the system info for C3 while using MS Edge. I don't know what all the info means so maybe there's a small difference that will stand out. I hope it proves useful.

Either way, thank you for your help as always.
Sincerely from across the Pond,
- Michael

Platform information
Browser: Edge
Browser version: 87.0.664.75
Browser engine: Chromium
Browser architecture: 64-bit
Context: browser
Operating system: Windows
Operating system version: 10
Operating system architecture: 64-bit
Device type: desktop
Device pixel ratio: 1
Logical CPU cores: 4
Approx. device memory: 8 GB
User agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.141 Safari/537.36 Edg/87.0.664.75
C3 release: r225 (stable)
Language setting: en-US

Local storage
Storage quota (approx): 142 gb
Storage usage (approx): 48 mb (0%)
Persistant storage: No

Browser support notes
This list contains missing features that are not required, but could improve performance or user experience if supported.

Nothing is missing. Everything is OK!
WebGL information
Version string: WebGL 2.0 (OpenGL ES 3.0 Chromium)
Numeric version: 2
Supports NPOT textures: yes
Supports GPU profiling: yes
Supports highp precision: yes
Vendor: Google Inc.
Renderer: ANGLE (Intel(R) HD Graphics 520 Direct3D11 vs_5_0 ps_5_0)
Major performance caveat: no
Maximum texture size: 16384
Point size range: 1 to 1024
Extensions:

EXT_color_buffer_float
EXT_color_buffer_half_float
EXT_disjoint_timer_query_webgl2
EXT_float_blend
EXT_texture_compression_bptc
EXT_texture_compression_rgtc
EXT_texture_filter_anisotropic
EXT_texture_norm16
KHR_parallel_shader_compile
OES_texture_float_linear
WEBGL_compressed_texture_s3tc
WEBGL_compressed_texture_s3tc_srgb
WEBGL_debug_renderer_info
WEBGL_debug_shaders
WEBGL_lose_context
WEBGL_multi_draw
OVR_multiview2
Audio information
System sample rate: 48000 Hz
Output channels: 2
Output interpretation: speakers
Supported decode formats:

WebM Opus (audio/webm; codecs=opus)
Ogg Opus (audio/ogg; codecs=opus)
WebM Vorbis (audio/webm; codecs=vorbis)
Ogg Vorbis (audio/ogg; codecs=vorbis)
MPEG-4 AAC (audio/mp4; codecs=mp4a.40.5)
MP3 (audio/mpeg)
FLAC (audio/flac)
PCM WAV (audio/wav; codecs=1)
Supported encode formats:

WebM Opus (audio/webm; codecs=opus)
Video information
Supported decode formats:

WebM AV1 (video/webm; codecs=av01.0.00M.08)
MP4 AV1 (video/mp4; codecs=av01.0.00M.08)
WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)
Ogg Theora (video/ogg; codecs=theora)
H.265 (video/mp4; codecs=hev1.1.2.L93.B0)
H.264 (video/mp4; codecs=avc1.42E01E)
Supported encode formats:

WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)
